src/storage/admin/DropCheckpointProcessor.cpp

```diff
@@ -13,7 +13,7 @@ void DropCheckpointProcessor::process(const cpp2::DropCPRequest& req) {
   auto spaceIdList = req.get_space_ids();
   auto& name = req.get_name();
   for (auto spaceId : spaceIdList) {
-    auto code = env_->kvstore_->dropCheckpoint(spaceId, std::move(name));
```

Comment:
Em, IIRC, in the old code, the second parameter in dropCheckpoint is copied not moved from the `name`. Are you sure this PR fix some bugs?

Reply:
I will test it

Reply:
My fault

First `name` is a const reference
Secondly, the second parameter of `dropCheckpoint` is also a const reference.
move is meaningless here.